_posts/2018-12-03-Reading: 7-reasons-why-smart,hardworking-people-don't-become-successful.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "2018-12-02-Reading: 7 reasons why smart, hardworking people don't become successful"
+date:   2018-12-02 19:31:07 +0800
+categories: readingnote
+---
+
+[Video Link](2018-12-02-Reading: https://www.youtube.com/watch?v=cqINcpbcZqs)
+
+#### Summary:
+1、You don't reach out to new people.
+2、You are averse to change.
+3、You're not willing to take risks.
+4、You believe you deserve success based on credentials.
+5、You constantly go after whatever's exciting at the moment.
+6、You can't commit to a decision.
+7、You don't believe in yourself.
+
+#### Reflection:
+The video is really refreshing and I think there are really some points I need to improve.
+First, make new friends especially from different area, have different life experience or have different ideas. Start by say hi to strangers, welcome different ideas, inspects, perspective etc.
+Second, go to different public occasion, try out new things, go to have much different life experience.
+Third, take risks, dont fear risks, when you take more responsibility it always means you will learn more and able to take more responsibility. Look at now from 10 years later, you will always grateful for the biggest risk you have taken, so have more courage now, dont be feared, there are really nothing to fear.
```

Comment:
```Thirdly```, take risks ```and don't``` fear risks```. When``` you take more responsibility```,``` it ```means you will always learn more``` and ```be``` able to take ```even``` more responsibility. ```Looking back in 10 years time```, you will ```be``` grateful for the biggest ```risks``` you have taken. So ```be more courageous``` now, ```don't be afraid``` (feared has a totally different meaning), there are really nothing to fear.